src/braket/experimental/autoqasm/api.py

```diff
@@ -193,7 +193,9 @@ def _convert(
         kwargs (Dict[str, Any]): Keyword arguments passed to the program when called.
 
     Returns:
-        Program: The converted program.
+        Union[Program, Optional[Var]]: The converted program, if this is a top-level call
+        to the conversion process. Or, the oqpy variable returned from the converted function,
+        if this is a subroutine conversion.
```

Comment:
Super nit, should these be hanging indent? I think we've been inconsistent about it. I guess it's not so important with a return value, cause there aren't other values to delineate from